### DISCUSSION TITLE
How can I change max_file_uploads in php.ini

### DISCUSSION THREAD

**Discussion by akiftezcan38** [wardenenv/warden#632] [Help & Support]
How can I change max_file_uploads in php.ini ?
I get this error. 413 Request Entity Too Large

post_max_size , max_file_uploads vs.

**Comment by atty31** [comment]
I m getting the same, have you find a fix ?

**Comment by sprankhub** [comment]
Create a file `.warden/warden-env.yml`:

	services:
	  nginx:
		volumes:
		  - ./.warden/nginx/custom.conf:/etc/nginx/default.d/custom.conf

Create a file `.warden/nginx/custom.conf`:

    client_max_body_size 256M;

Run `warden env up --force-recreate`.
